enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = "3.2.14"
+__version__ = "3.2.14.post1"
```

Comment:
```suggestion
__version__ = "3.2.14.pearson.juniper"
```
What do you think? @diegomillan

Reply:
I'm sticking to [PEP-440](https://www.python.org/dev/peps/pep-0440/#post-releases)

> Within a numeric release (1.0, 2.7.3), the following suffixes are permitted and MUST be ordered as shown:

.devN, aN, bN, rcN, <no suffix>, .postN

It was also tested executing the `pip install` command